source/common/http/conn_pool_grid.cc

```diff
@@ -185,17 +185,22 @@ ConnectivityGrid::ConnectivityGrid(
     const Network::ConnectionSocket::OptionsSharedPtr& options,
     const Network::TransportSocketOptionsConstSharedPtr& transport_socket_options,
     Upstream::ClusterConnectivityState& state, TimeSource& time_source,
-    AlternateProtocolsCacheSharedPtr alternate_protocols,
-    std::chrono::milliseconds next_attempt_duration, ConnectivityOptions connectivity_options,
+    AlternateProtocolsCacheSharedPtr alternate_protocols, ConnectivityOptions connectivity_options,
     Quic::QuicStatNames& quic_stat_names, Stats::Scope& scope)
     : dispatcher_(dispatcher), random_generator_(random_generator), host_(host),
       priority_(priority), options_(options), transport_socket_options_(transport_socket_options),
-      state_(state), next_attempt_duration_(next_attempt_duration), time_source_(time_source),
-      http3_status_tracker_(dispatcher_), alternate_protocols_(alternate_protocols),
-      quic_stat_names_(quic_stat_names), scope_(scope) {
+      state_(state), next_attempt_duration_(std::chrono::milliseconds(300)),
```

Comment:
total up to you: consider factoring this out into something like kDefaultFailoverTime, or some such. (Or not)

Reply:
done!